src/prerender.ts

```diff
@@ -1,9 +1,11 @@
 import path from "node:path";
-
 import { promises as fs } from "node:fs";
 
 import MagicString from "magic-string";
 import { parse as htmlParse } from "node-html-parser";
+import { SourceMapConsumer } from "source-map";
+import { parse as StackTraceParse } from "stack-trace";
+import { codeFrameColumns } from "@babel/code-frame";
```

Comment:
I was originally using `simple-code-frame`, but Babel here comes for free as we're already using it for transpiling and whatnot.

Reply:
yeah babel's code frame also supports basic syntax highlighting which is pretty nice!

Reply:
Ah, well unfortunately that's not usable here. Not sure if it's Rollup or Vite, but the entire message is a flat red.

Reply:
![temp](https://github.com/preactjs/preset-vite/assets/33403762/2f60e732-ea65-48b9-8c96-804d4c499280)